skxray/core.py

```diff
@@ -603,27 +603,22 @@ def bin_1D(x, y, nx=None, min_x=None, max_x=None):
     return bins, val, count
 
 
-def pixel_to_radius(shape, calibrated_center, pixel_size=None):
+def radial_grid(center, shape, pixel_size=None):
```

Comment:
Can you document this in https://github.com/Nikea/scikit-xray/tree/master/doc/api_changes ?